frontend/src/features/public-form/utils/createSubmission.ts

```diff
@@ -89,7 +89,7 @@ const createResponsesArray = (
 ): FieldResponse[] => {
   const transformedResponses = formFields
     .map((ff) => transformInputsToOutputs(ff, formInputs[ff._id]))
-    .filter((output): output is FieldResponse => output !== undefined)
+    .filter((output): output is FieldResponse => output !== null)
```

Comment:
to check - what prompted this change?

Reply:
I totally do not recall this change but thank you for pointing this out. Let me investigate.

Reply:
`git blame` is at [Kar Rui](https://github.com/opengovsg/FormSG/commit/4b7c16fa2174e865ed85b82f6146a5de7b67d840)